### DISCUSSION TITLE
Python codegen: getting multiple functions in __init__

### DISCUSSION THREAD

**Discussion by jpreiss** [symforce-org/symforce#393] [Q&A]
I am doing python codegen for several functions with Jacobians into the same directory. If I run codegen for functions `a`, `b`, and `c`, I am left with an `__init__.py` file that only contains:

    from .c import c

How do I make symforce include multiple functions in the `__init__`?

Sorry if this is obvious, but I couldn't find it in the tutorial or the `PythonConfig` doc.

Thanks for this super cool and useful project! I hope to share a new use case soon :)

**Comment by aaron-skydio** [comment]
Sorry for the delay responding!  This is https://github.com/symforce-org/symforce/issues/21, although I wouldn't have expected you to find that - will have a fix shortly
